src/x/process/process_notlinux.go

```diff
@@ -29,7 +29,20 @@ import (
 var errNotAvailable = errors.New(
 	"cannot get process file descriptors, only available on linux")
 
-// NumFDs returns the number of file descriptors for a given process.
+// NumFDs returns the number of file descriptors for a given process and is not available
+// on non-linux systems.
 func NumFDs(pid int) (int, error) {
 	return 0, errNotAvailable
 }
+
+// NumFDsWithDefaultBatchSleep returns the number of file descriptors for a given process
+// and is not available on non-linux systems.
+func NumFDsWithDefaultBatchSleep(pid int) (int, error) {
```

Comment:
Don't you also need `NumFDsReference` (or whatever that gets renamed to) from the non-linux source file?  Otherwise it won't be able to build on non-linux since it will be missing.

Reply:
ended up not exporting it